src/test/java/io/obswebsocket/community/test/translator/requestSerializationTests/GeneralRequestsSerializationTest.java

```diff
@@ -0,0 +1,185 @@
+package io.obswebsocket.community.test.translator.requestSerializationTests;
+
+import com.google.gson.JsonObject;
+import io.obswebsocket.community.message.Message;
+import io.obswebsocket.community.message.request.Request;
+import io.obswebsocket.community.message.request.general.*;
+import io.obswebsocket.community.model.Projector;
+import io.obswebsocket.community.test.translator.AbstractSerializationTest;
+import io.obswebsocket.community.translator.GsonMessageTranslator;
+import io.obswebsocket.community.translator.MessageTranslator;
+import org.json.JSONException;
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Fail.fail;
+
+public class GeneralRequestsSerializationTest  extends AbstractSerializationTest {
+    @Test
+    void broadcastCustomEventRequest() {
+        JsonObject eventData = new JsonObject();
+        eventData.addProperty("customEventType", "customEvent");
+        eventData.addProperty("boolean", true);
+        eventData.addProperty("integer", 10);
+
+        BroadcastCustomEventRequest broadcastCustomEventRequest = BroadcastCustomEventRequest.builder().requestData(eventData).build();
+
+        String json = "{\n" +
+                "\t\"requestData\": {\n" +
+                "\t\t\"customEventType\": \"customEvent\",\n" +
+                "\t\t\"boolean\": true,\n" +
+                "\t\t\"integer\": 10\n" +
+                "\t},\n" +
+                "\t\"requestType\": \"BroadcastCustomEvent\",\n" +
+                "\t\"requestId\": " + broadcastCustomEventRequest.getRequestId() + ",\n" +
+                "\t\"messageType\": \"Request\"\n" +
+                "}";
+
+        MessageTranslator translator = new GsonMessageTranslator();
```

Comment:
Why are we using a different way to test for this request?

Reply:
Because when Gson deserializes the json it gets the Integer as a json Primitive and that made the test fail. @TinaTiel told me to use this method instead.

Reply:
I see, let me get back to you

Reply:
Found nothing for now, let's move on